backend/src/graphql/resolver/UserResolver.test.ts

```diff
@@ -723,6 +725,7 @@ describe('UserResolver', () => {
       })
 
       it('throws an error', async () => {
+        jest.clearAllMocks()
```

Comment:
What's the reason to clearAllMocks here?
There is no logger.test after the testcase.

Reply:
same